student.rb

```diff
@@ -1,8 +1,11 @@
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission: parent_permission)
     @parent_permission = parent_permission
     @classroom = classroom
+    classroom.add_student(self)
   end
 
```

Comment:
- Hey, let's add establish the relationship as a student belongs to a classroom by adding the method to establish that below


```
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```

Reply:
Done!

Reply:
- [ ] Please add a new method that serves as the 'belongs-to' side (a student belongs to a classroom)
```
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
  
```